crypto/src/hash/rescue/rp64_256/mds_freq.rs

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source &code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+// FFT MDS MULTIPLICATION HELPER FUNCTIONS
+// --------------------------------------------------------------------------------------------
+// MDS
```

Comment:
nit: I'd probably get rid of these two lines.

Reply:
Thank you, I have merged the commits that were ahead. I hope it works now.